src/mithril.postgrest.js

```diff
@@ -1,12 +1,12 @@
 (function (factory) {
     if (typeof exports === 'object') {
         // Node/CommonJS
-        factory(require('mithril'), require('underscore'), require('node-localstorage'));
+        factory(require('mithril'), require('node-localstorage'));
     } else {
         // Browser globals
-        factory(m, _, localStorage);
```

Comment:
To remove the underscore dependency we would have to replace the _.compose, _.extend and _.isFunction calls.